Projects/Feature/SigninFeature/Sources/Scene/SigninView.swift

```diff
@@ -0,0 +1,48 @@
+import SwiftUI
+import BaseFeature
+import DesignSystem
+import GlobalThirdPartyLibrary
+import ViewUtil
+import UtilityModule
+
+struct SigninView: View {
+    @StateObject var container: MVIContainer<SigninIntentProtocol, SigninStateProtocol>
+    var intent: any SigninIntentProtocol { container.intent }
+    var state: any SigninStateProtocol { container.model }
+
+    var body: some View {
+        VStack(spacing: 0) {
+            Text("Student\nManagement\nService")
```

Comment:
```suggestion
            Text("STUDENT\nMANAGEMENT\nSERVICE")
```
Text 내용물 이거로 바꿔야할거같아요